### ISSUE TITLE
Playback Accessibility

### ISSUE THREAD

**Issue by poffensand** [polimediaupv/paella#633]
Playback of a video in paella player cannot be started via screenreader. To clarify, once playback started, screenreader works without problems on all neccessary controls, it is only before playback has been started and the preview image is shown that playback cannot be started by a screenreader (clickable preview image is not targetable).

# Environment Information
 - Paella version: as being shipped with Opencast 8.7
 - If inspected with Mozilla Firefox "Inspect Accessibility Properties", following message is shown: "Clickable elements must be focusable and should have interactive semantics"
